public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -918,22 +919,19 @@ export function formValuesFromExistingRule(rule: RuleWithLocation<RulerRuleDTO>)
 }
 
 export function getInstantFromDataQuery(model: AlertDataQuery, type: string): boolean | undefined {
-  // if the datasource is not prometheus or loki, instant is defined in the model or defaults to undefined
-  if (type !== DataSourceType.Prometheus && type !== DataSourceType.Loki) {
-    if ('instant' in model) {
-      return model.instant;
-    } else {
-      if ('queryType' in model) {
-        return model.queryType === 'instant';
-      } else {
-        return undefined;
-      }
-    }
+  if (!type) {
```

Comment:
Args in function signature is: `query: AlertQuery<AlertDataQuery>` in [main](https://github.com/grafana/grafana/blob/b39eaac69e78187a313b050b8cea9cedbbe47d43/public/app/features/alerting/unified/utils/rule-form.ts#L872-L898) compared to `model: AlertDataQuery, type: string` here.

Reason for change: the signature update was a [recent change](https://github.com/grafana/grafana/pull/102119/files#diff-0c75a47c20ee2dedbfbc9baa8629de430ab5f3e6ad0bf1ed091d990a84a34e59) that did not make it into 11.5.x. I kept the function signature as is in the 11.5.x branch so that the code calling `getInstantFromDataQuery` would not have to be changed.